lib/bojangles.rb

```diff
@@ -65,13 +65,15 @@ def get_avail_departure_times!
 
   def go!
     error_messages, statuses = DepartureComparator.compare
-
+    time_occurred = Time.now
+    
     if error_messages.present?
       MAIL_SETTINGS[:html_body] = message_html(error_messages)
       if CONFIG['environment'] == 'development'
         MAIL_SETTINGS.merge! via: :smtp, via_options: { address: 'localhost', port: 1025 }
       end
       Pony.mail MAIL_SETTINGS
+      update_log_file! to: time_occurred, error_messages, statuses
```

Comment:
You're correct, but I'm asking whether this is valid Ruby syntax, and if so whether it works as intended. If so, I'm not sure how.